### DISCUSSION TITLE
Is there a way to only set minSize to a single side?

### DISCUSSION THREAD

**Discussion by klicpjan** [nathancahill/split#549] [Q&A]
Hello, I was wondering whether there is a way to easily set minSize to only one side of the gutter. I am using split-grid. I need to have a resizable sidebar, but I want to limit the size this sidebar can be expanded by - the sidebar is on the right, hence I would like to set min size to the left side of the gutter, but no min size to the right side. Is there perhaps some better way to do this, rather than using the minSize property? Many thanks!

**Comment by bsastregx** [comment]
I think that if you dont want a min size to the right side, you just pass 0 as the argument, for the right side.
